### ISSUE TITLE
Showing friends doesn't work in both directions

### ISSUE THREAD

**Issue by mlandauer** [openaustralia/publicwhip#280]
This is pretty easy to replicate.

Go to Tony Abbott's page with all his friends: http://publicwhip-rails.openaustraliafoundation.org.au/mp.php?mpn=Tony_Abbott&mpc=Warringah&house=representatives&display=allfriends

His top friend is Bruce Scott. If you go to his page with all his friends: http://publicwhip-rails.openaustraliafoundation.org.au/mp.php?mpn=Bruce_Scott&mpc=Maranoa&house=representatives&display=allfriends

No sign of Tony Abbott at all in the list of friends! Eek.

In other words this list is completely wrong.

Also, I can reproduce the same problem on the php version.
